e2e/playwright/fixtures/sceneFixture.ts

```diff
@@ -78,6 +91,47 @@ export class SceneFixture {
         return this.page.mouse.move(x, y, { steps })
       },
     ] as const
+  makeDragHelpers = (
```

Comment:
Tried to follow the pattern used with `makeMouseHelpers` move/click fixture helper. I found adding to the returned array clunky if the developer only wanted dragging handlers, so I broke this out on its own.